script.js

```diff
@@ -13,5 +14,284 @@ document.querySelectorAll(".nav-link").forEach(n => n.addEventListener("click",
     navMenu.classList.remove("active");
 }))
 
+/**************** Refactoring projects section ****************/
+let projects = [
+    {
+        id: 1,
+        title: "Multi-Post Stories Gain+Glory",
+        languages: ["Ruby on Rails", "CSS", "Javascript", "HTML"],
+        buttonName: "See Project"
+    },
 
+    {
+        id: 2,
+        title: "Multi-Post Stories Gain+Glory",
+        languages: ["Ruby on Rails", "CSS", "Javascript", "HTML"],
+        buttonName: "See Project"
+    },
 
+    {
+        id: 3,
+        title: "Multi-Post Stories Gain+Glory",
+        languages: ["Ruby on Rails", "CSS", "Javascript", "HTML"],
+        buttonName: "See Project"
+    },
+
+    {
+        id: 4,
+        title: "Multi-Post Stories Gain+Glory",
+        languages: ["Ruby on Rails", "CSS", "Javascript", "HTML"],
+        buttonName: "See Project"
+    },
+
+    {
+        id: 5,
+        title: "Multi-Post Stories Gain+Glory",
+        languages: ["Ruby on Rails", "CSS", "Javascript", "HTML"],
+        buttonName: "See Project"
+    },
+
+    {
+        id: 6,
+        title: "Multi-Post Stories Gain+Glory",
+        languages: ["Ruby on Rails", "CSS", "Javascript", "HTML"],
+        buttonName: "See Project"
+    },
+]
+
+/* The cards container */
+let cards = document.querySelector('.cards');
+
+/* Create cards */
+projects.forEach((project) => {
+    /* Create the elements */
+    let div = document.createElement('div');
+    let divImg = document.createElement('div');
+    let cardTextDiv = document.createElement('div')
+    let h3 = document.createElement('h3');
+    let ul = document.createElement('ul');
+    let button = document.createElement('button');
+
+    /* Giving classes */
+    div.classList.add('card');
+    divImg.classList.add('card-image');
+    cardTextDiv.classList.add('card-description');
+    h3.classList.add('card-title');
+    ul.classList.add('card-tools-list');
+    button.classList.add('see-project', 'open-modal');
+
+    /* Assing values */
+    h3.textContent = project.title;
+    button.textContent = project.buttonName;
+    button.dataset.projectId = project.id;
+
+    /* Creating languages list */
+    project.languages.forEach((language) => {
+        let li = document.createElement('li')
+        li.classList.add('card-tool')
+        li.textContent = language;
+        ul.appendChild(li);
+    });
+
+    /* Append elements to cardTextdiv */
+    cardTextDiv.appendChild(h3);
+    cardTextDiv.appendChild(ul);
+    cardTextDiv.appendChild(button);
+
+    /* Append elements to the div */
+    div.appendChild(divImg);
+    div.appendChild(cardTextDiv);
+
+    /* Append card div to the cards container */
+
+    cards.appendChild(div);
+});
+
+/****************** popup window *****************/
+const popups = [
```

Comment:
- [Optional] Ideally, your project cards should have images for each of the projects and you shouldn't have a different array to store pop-ups. The project array should contain all the project details, but the project card is only displaying some of the details. When a project is clicked, that's when a pop-up will appear to display more information about the project from the same project array. This will make your code drier and also when you are adding a new project, you don't need to add to both the popup array and the project array.